HISTORY.rst

```diff
@@ -2,6 +2,16 @@
 History
 =======
 
+0.7.2 (2025-03-07)
+------------------
+* Fix: `pyfar.dsp.find_impulse_response_delay` crashed when the input impulse response had a large negative peak. This is now fixed (PR #776)
+* Fix: Add missing setter for the state, which accepts array likes, in all filter classes (PR #757)
+* Fix: `pyfar.dsp.find_impulse_response_start` now returns an array of integers as described in the documentation. Before an array of floats was returned (PR #764)
+* Fix: Disable the possibility to set x, y, and z coordinates using `coordinates.x[0] = 1`.  This is now consistent with all other coordinates (PR #778)
+* Fix: `pyfar.Coordinates.find_within` now correctly works for multi-dimensional Coordinates objects. This was fixed by flattening the radii to check if all radii before checking if all radii are the same (PR #742)
```

Comment:
Something is wrong with this sentence, isn't it?

Reply:
Yes, fixed now. Also releasing now :)